Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -5616,6 +5616,24 @@ public static void ListEntitlements(Action<LootLockerEntitlementHistoryResponse>
         {
             ListEntitlements(-1, null, onComplete);
         }
+
+        /// <summary>
+        /// Get a single entitlement
```

Comment:
Not enough of an explanation imo. This relies on people having read the documentation of ListEntitlements before reading this one. I would add some brief info on what an entitlement is.